### ISSUE TITLE
Feature request: WaveNetVA implementation

### ISSUE THREAD

**Issue by 38github** [Chowdhury-DSP/chowdsp_utils#361]
I would really like this option since an other plugin supporting this is buggy and you might even manage to get the CPU down like you did with Proteus/GuitarML AND it will be available as CLAP. I will donate to the project and hopefully others will do the same.

**Comment by jatinchowdhury18** [comment]
Yeah, having a generic and performance WaveNetVA implementation would be pretty nice! However, I think that sort of thing would fit better in the [RTNeural](https://github.com/jatinchowdhury18/RTNeural) library. If it's okay with you, I can move the issue over there?

**Comment by 38github** [comment]
Of course, it is alright. You are the developer and knows better than me so go on. WaveNet recreates profiles/models so much better than any LSTM implementation I have heard but it uses a lot more CPU. I have made models with GuitarML's Colab scripts which I can't distinguish from the original. The problem with SmartPedal though is that the Linux version crashes all the time on all my systems. The upside if you implement it, besides probably less CPU usage, is that we will have a CLAP version and a plugin that is stable.
